global.R

```diff
@@ -121,10 +121,23 @@ doseTableInit <- data.frame(
 )
 doseTableNewRow <-  doseTableInit[5, ]
 
+eventTableInit <- data.frame(
+  Time = 0,
+  Event = "",
+  Fill = "",
+  stringsAsFactors = FALSE
+)[FALSE, ]
+
 `%then%` <- shiny:::`%OR%`
 
-outputComments <- function(text, echo = TRUE) {
+outputComments <- function(..., echo = TRUE) {
   isolate({
+    argslist <- list(...)
+    if (length(argslist) == 1) {
+      text <- argslist[[1]]
+    } else {
+      text <- paste(argslist, collapse = "")
+    }
```

Comment:
I made `outputComments()` to work with any number of arguments, so instead of doing `outputComments(paste("hello", "there"))` you can just do `outputComments("hello", "there")`